terasoluna-gfw-libraries-parent/pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>terasoluna-gfw-libraries-parent</artifactId>
+    <packaging>pom</packaging>
+    <name>TERASOLUNA Server Framework for Java (5.x) Developer Tools</name>

```

Comment:
@btyoshikawaa this comment correspond to f203fe4d1b30478357c925eba5000421bf73c27e.
I renamed this name `Common Libraries` likewise `terasoluna-gfw-dependencies`'s name